docpages/example_programs/the_basics/using_callback_functions.md

```diff
@@ -0,0 +1,97 @@
+\page callback-functions Using Callback Functions
+
+When you create or get an object from Discord, you send the request to its API and in return you get either an error or the object you requested/created. Let's create a message and get it:
+
+~~~~~~~~~~~~~~{.cpp}
+#include <dpp/dpp.h>
+#include <fmt/format.h> // used to format text, install and define FMT_HEADER_ONLY to use
+
+struct GetMessage {
```

Comment:
What is this for? It doesn't seem to be used by anything else
D++ convention is snake_case as well so that should be `get_message` if it is to stay

Reply:
I just wanted to create a struct containing ID's used in the test. Should I not use the struct?

Reply:
I recommend finding a way that doesn't require using specific IDs in the code yes

Reply:
Sure!

Reply:
I think I could replace message_get with messages_get